### PR TITLE
check a object is empty

### PR DESCRIPTION
**Pull request by Provenr**

### CODE REVIEW

**Comment by phuocng** [comment]
Thanks @Provenr. We already had the [Check if an object is empty](https://1loc.dev/#check-if-an-object-is-empty) snippet.

File: collections/function/check-params-object-type-of-function.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+~~~ javascript
+const Type = object => Object.prototype.toString.call(object).match(/\[object (.*)\]/)[1]
```
Comment (phuocng) [inline collections/function/check-params-object-type-of-function.md]: Can you change the function name to `getTypeOf`?

Comment (Provenr) [reply collections/function/check-params-object-type-of-function.md]: sure

File: collections/function/check-params-object-type-of-function.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Type(null)                            // Null
+Type(undefined)                       // Undefined
+Type({})                              // Object
+Type([])                              // Array
+Type(/[a-z]/g)                        // RegExp
+Type(new Date(2020))                  // Date
+Type(new Error())                     // Error
+Type(function(){})                    // Function
+Type(document)                        // HTMLDocument
+~~~
```
Comment (phuocng) [inline collections/function/check-params-object-type-of-function.md:19]: Can you add examples for other types, such as an arrow function, `async`, `await`?

Comment (Provenr) [reply collections/function/check-params-object-type-of-function.md:19]: OK

File: collections/validator/check-if-a-object-is-empty.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+~~~ javascript
+const isEmpty = object => Object.keys(object).length === 0
+
+// Examples
+isEmpty({a:1});      // false
+isEmpty({});         // true
+issEmpty({x: undefined})  // true
+~~~
```
Comment (phuocng) [inline collections/validator/check-if-a-object-is-empty.md]: Can you remove this file as we already had it at https://1loc.dev/#check-if-an-object-is-empty?

File: collections/function/check-params-object-type-of-function.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+getTypeOf(null)                            // Null
+getTypeOf(undefined)                       // Undefined
+getTypeOf({})                              // Object
+getTypeOf([])                              // Array
+getTypeOf(/[a-z]/g)                        // RegExp
+getTypeOf(new Date(2020))                  // Date
+getTypeOf(new Error())                     // Error
+getTypeOf(function(){})                    // Function
+getTypeOf(async ()=>{})                    // Function
+Type(async ()=>{})                         // AsyncFunction
```
Comment (phuocng) [inline collections/function/check-params-object-type-of-function.md]: I think you forget to remove this line.

File: collections/validator/check-if-a-object-is-empty.md (hunk 3 above)
Comment (Provenr) [reply collections/validator/check-if-a-object-is-empty.md]: OK

**Comment by phuocng** [comment]
@Provenr The snippet is available at https://1loc.dev/#get-type-of-a-variable-in-string

Looking forward to seeing more from you :+1:

**Pull request merged**